src/AutoRest.CSharp/DataPlane/Generation/DataPlaneClientWriter.cs

```diff
@@ -142,7 +142,7 @@ private void WriteClientCtors(CodeWriter writer, DataPlaneClient client, BuildCo
             var clientOptionsName = ClientBuilder.GetClientPrefix(context.DefaultLibraryName, context);
             foreach (var scheme in schemes)
             {
-                if (scheme is AzureKeySecurityScheme azureKeySecurityScheme)
+                if (scheme is AzureKeySecurityScheme || scheme is KeySecurityScheme)
```

Comment:
I think we should throw exception on `AzureKeySecurityScheme` and `AADTokenSecurityScheme` cause we don't expect old types to be produced by M4.